change/@microsoft-teams-js-355651b0-c6d4-47bb-b923-426b9a9a3bae.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Explicitly use browser implementation of debug to resolve polyfill issue.",
```

Comment:
> polyfill issue [](http://example.com/codeflow?start=71&length=15)

Thinking about how someone not involved might read this in the changelog, is it worth calling out that it was a `process` polyfill issue? Or was the issue more broad than just that one? Or would devs not have seen "process" listed in the error message so it doesn't matter if you call it out here?

Reply:
This bug actually manifested itself in a couple different ways, that on first glance would seem unrelated, so I figured it was best to just keep it generic. I'm not really sure there is a concise way of covering all manifestations of the bug that would be appropriate for a changelog.